Dev/VideoChatClient/Output/ImageOutput.cpp

```diff
@@ -0,0 +1,16 @@
+#include "ImageOutput.hpp"
+
+#include "VideoChatClientException/VideoChatClientException.hpp"
+
+void ImageOutput::show_output(Buffer& out)
+{
+	try
+	{
+		cv::Mat img = cv::imdecode(out, cv::IMREAD_COLOR);
+		cv::imshow("Image", img);
+	}
+	catch (...)
```

Comment:
let me know the reason for the try catch

Reply:
Code can fail in case cv2 does not detect an image in the bytes given.